### PR TITLE
[Merged by Bors] - Bump kube to 0.78.0 and k8s-openapi to 0.17.0. Bump k8s version

### PR DESCRIPTION
**Pull request by sbernauer**
## Review Checklist
- [ ] Code contains useful comments
- [ ] (Integration-)Test cases added (or not applicable)
- [ ] Documentation added (or not applicable)
- [ ] Changelog updated (or not applicable)
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: src/builder/pod/container.rs [hunk 1]
```diff
@@ -359,6 +359,7 @@ mod tests {
                 ]
                 .into(),
             ),
+            claims: None,
```
Comment (maltesander) [inline src/builder/pod/container.rs:362]: For tests i would default:
```suggestion
            ..ResourceRequirements::default()
```

**Review by maltesander — APPROVED** [review]
Thanks! LGTM, one minor thing.

File: src/builder/pod/container.rs (hunk 1 above)
Comment (sbernauer) [reply src/builder/pod/container.rs:362]: Makes sense

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,8 +9,10 @@ All notable changes to this project will be documented in this file.
 ### Changed
 
 - Disable Vector agent by default ([#526]).
+- Bump kube to 0.78.0 and k8s-openapi to 0.17.0. Bump k8s version from 1.24 to 1.26 ([#532]).
 
 [#526]: https://github.com/stackabletech/operator-rs/pull/526
+[#532]: https://github.com/stackabletech/operator-rs/pull/532
```
[the remark this answers is not in the document]
Comment (sbernauer) [reply CHANGELOG.md:15]: Arrrghhhh

**Pull request closed without merging**